build.gradle.kts

```diff
@@ -35,6 +35,7 @@ dependencies {
     compileOnly("org.projectlombok:lombok")
     annotationProcessor("org.projectlombok:lombok")
 
+    implementation("org.springframework.boot:spring-boot-starter-validation:3.4.0")
```

Comment:
Isn't the same version as spring-boot?

Reply:
I used the latest version, but can cause issues so removed the version now should drive from the spring-boot plugin